addons/l10n_ch/data/account.account.template.csv

```diff
@@ -141,8 +143,30 @@
 "ch_coa_4903","Commissions on purchases","4903","expense","l10n_ch.l10nch_chart_template","False"
 "ch_coa_4906","Exchange rate differences","4906","expense","l10n_ch.l10nch_chart_template","False"
 "ch_coa_5000","Wages and salaries","5000","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5003","Salary Advance","5003","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5004","Holidays payment after departure","5004","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5005","13th Month","5005","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5006","Gratification","5006","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5007","Bonus","5007","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5009","Jubilee Gift","5009","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5030","Travelling Expenses","5030","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5040","Salary Allowances","5040","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5601","CA Fees","5601","expense","l10n_ch.l10nch_chart_template","False"
 "ch_coa_5700","Social benefits","5700","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5701","Social benefits AC","5701","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5720","Social Benefits LPP","5720","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5721","Social Benefits LPP Redemption","5721","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5730","Social Benefits AANP","5730","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5731","Social Benefits Optional LAAC Redemption","5731","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5740","Social Benefits IJM/CM","5740","expense","l10n_ch.l10nch_chart_template","False"
+"ch_coa_5790","Withholding Taxes IS","5790","expense","l10n_ch.l10nch_chart_template","False"
```

Comment:
Are we sure `"Withholding Taxes IS"` is an expense account? (and thus by extension, is it the right code?)

From my experience, those are more likely to be of type `liability_current` or `asset_current`.

Also, from what I read here: https://www.taxolution.ch/swiss-tax-guide/withholding-tax-in-switzerland/

It's an expense for the employee not for the company.

I think It's more likely this account will be one of the credited account with when you debit the expense account `"Wages and salaries","5000"`.